diagrams/generate.dart

```diff
@@ -469,5 +493,6 @@ Future<Null> main(List<String> arguments) async {
     keepTmp = true;
   }
 
-  new DiagramGenerator(temporaryDirectory: temporaryDirectory, cleanup: !keepTmp)..generateDiagrams();
+  new DiagramGenerator(temporaryDirectory: temporaryDirectory, cleanup: !keepTmp)
```

Comment:
we should rename "keepTmp" to "keepTemporaryDirectory" while we're at it

Reply:
Done